lms/djangoapps/django_comment_client/utils.py

```diff
@@ -132,7 +132,9 @@ def get_accessible_discussion_xblocks(course, user, include_all=False):  # pylin
     Return a list of all valid discussion xblocks in this course that
     are accessible to the given user.
     """
-    all_xblocks = modulestore().get_items(course.id, qualifiers={'category': 'discussion'}, include_orphans=False)
+    all_xblocks = modulestore().get_items(course.id, qualifiers={
+        'category': ['discussion', 'discussion-forum']
```

Comment:
This change is stopping any previously showing discussions to not show up for me.
Are xblock qualifier lists AND lists or OR lists? If it's an AND list, one solution would be to [pass a regex object](https://github.com/edx-solutions/edx-platform/blob/a869ae88e2554961c74132fc2c66598a11420c7e/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L1069) as the qualifier that matches against both "discussion" and "discussion-forum".

Worst case scenario is that it's necessary to do two calls to `get_items`, specifying different categories for each..